app/routes/reminders/reminder.js

```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+
+  model(params) {
+    return this.get('store').peekRecord('reminder', params.reminder_id)
```

Comment:
Dig it :shallow_pan_of_food: (ps that emoji is called "shallow pan of food"...wtf